components/commons/InputTitle.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from '@emotion/styled';
+
+type InputTitleProps = {
+  children: React.ReactNode;
+};
+
+const InputTitle = ({ children }: InputTitleProps) => {
+  if (typeof children !== 'string') {
+    return null;
+  }
+  return <InputTitleWrapper>{children ? children : ''}</InputTitleWrapper>;
+};
+
+const InputTitleWrapper = styled.span({
```

Comment:
여기는 span 보다 label 으로 보는게 맞겠네요 ㅎㅎ htmlFor 프롭 추가해주면 좋을 것 같습니다.
```suggestion
const InputTitleWrapper = styled.label({
```